whitelabelco/whitelabel-components
# feat: Standardize commits & versioning

## Description
This PR:
- Adds `commitizen` to use as a way for standardizing the commits in an interactive manner
- Adds `standard-version` to semi automate the release process which does the below.
  - bumps version based on commits
  - adds package.json and commits it
  - creates a git tag for the release
- Adds Readme section for Release Management

## To Test :mag: 
- Try using the commands
  - `yarn commit`: To add a commit in an interactive way
  - `yarn prerelease:dry-run`: Shows the steps and the version the release will go to
  - `yarn prerelease`: Actually creates an alpha pre release version for testing
  - `yarn release`: Creates the final release after the alpha version is tested. 
- Read [Readme.md](https://github.com/whitelabelco/whitelabel-components/blob/94d015655167c7325eadb3ee17ef7a75d27e7573/README.md) and verify the language is good and process is accurately defined


## Checklist :white_check_mark:
- [x] I have re-assigned the Github ticket and moved the ticket on the project board.
- [x] If I have added an ENV var or other secrets, I have updated Dashlane and the ReadMe (and any other applicable docs)

## Additional Details and Screenshots :art:
